deblur/parallel_deblur.py

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015, The Deblur Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
```

Comment:
QIIME has a GPL license. Doesn't matter if people gave permission, the license has not changed.